.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: node_js
 node_js:
-  - '6'
-  - '4'
+  - '11'
```

Comment:
I don't test on unstable Node.js versions. They break all the time. We can start adding `'12'` here in a couple of weeks when it's out.

Reply:
I wanted to make sure that we test this even on unstable releases to have an early notice when something breaks. This lib depends on undocumented internals so I thought it might be worth it.